### PR TITLE
Update frequency-plans.md

### PR DESCRIPTION
**Pull request by aattila**
EU_433 frequency plan  proposal, based on a RAK811 (EU433) channel definition.

### CODE REVIEW

**Comment by htdvisser** [comment]
Thanks for the proposal, @aattila. We won't officially support EU433 in v2, but let's see if we can use this for v3.

Could you please provide us with the link/source of this RAK811 (EU433) channel configuration?

File: _content/lorawan/frequency-plans.md [hunk 1]
```diff
@@ -32,7 +32,20 @@ Downlink:
...
+3. **433.575** - SF7BW125 to SF12BW125
+4. **433.975** - SF7BW125 to SF12BW125
+5. **434.175** - SF7BW125 to SF12BW125
+6. **434.375** - SF7BW125 to SF12BW125
+7. **434.575** - SF7BW125 to SF12BW125
+8. **434.775** - SF7BW125 to SF12BW125
+
+Downlink:
+
+- Uplink channels 1-8 (RX1)
```
Comment (htdvisser) [inline _content/lorawan/frequency-plans.md:48]: We have 434.665MHz / DR0 (SF12, 125kHz) for RX2

**Comment by aattila** [comment]
The uplink info is coming from the serial connection to the RAK811 WisNode. 
When I execute the command: 
`at+get_config=lora:channel`
I get:
```
OK.Max_nb_chs=16:
* 0,on,433175000,0,5; * 1,on,433375000,0,5; * 2,on,433575000,0,5; * 3,on,433975000,0,5; * 4,on,434175000,0,5; * 5,on,434375000,0,5; * 6,on,434575000,0,5; * 7,on,434775000,0,5; 
 8,off,0,0,0;  9,off,0,0,0; 10,off,0,0,0; 11,off,0,0,0; 12,off,0,0,0; 13,off,0,0,0; 14,off,0,0,0; 15,off,0,0,0 
```

**Comment by johanstokking** [comment]
@tftelkamp what do you think?

**Comment by jpmeijers** [comment]
At a glance it looks like this will fit into the normal ISM band (CEPT/ERC/REC 70-03). Need to double check with some EU countries' regulations too, and look at what the overlap is with the RFID channels.

image

**Comment by htdvisser** [comment]
I pushed a frequency plan config for v3 here: https://github.com/TheThingsNetwork/lorawan-frequency-plans/compare/feature/eu-433. 

You should be able to test it on v3 deployments by setting `frequency-plans.config-source` to `url` and `frequency-plans.url` to `https://raw.githubusercontent.com/TheThingsNetwork/lorawan-frequency-plans/feature/eu-433`

**Comment by johanstokking** [comment]
@aattila @jpmeijers have you had a chance to test this?

**Comment by aattila** [comment]
I cannot test in a near future.

**Comment by johanstokking** [comment]
OK we'll pick this up as part of github.com/TheThingsNetwork/lorawan-frequency-plans in the future, as we need technical support first before documenting it.

**Pull request closed without merging**